angular/src/app/login.service.ts

```diff
@@ -3,7 +3,7 @@ import { HttpClient, HttpResponse } from '@angular/common/http';
 import { catchError, filter, map } from 'rxjs/operators';
 import { Observable, of, throwError } from 'rxjs';
 
-@Injectable()
+@Injectable({ providedIn: 'root' })
 export class LoginService {
   constructor(private readonly http: HttpClient) { }
```

Comment:
Hi @jordanpowell88 
Refering to https://github.com/cypress-io/cypress/issues/23427#issuecomment-1224815367
Of course you can still override this LoginService to inject a double inside an Angular CT this way. However, it's only possible to override it for the very exact component under test. You can neither inject a double for a child component, nor can you inject a double to be used as a dependency for any other singleton service. For example: how would you inject a double for HttpClient as it is required by LoginService?